packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -651,6 +684,19 @@ export default class TabDisks extends Component {
     (vdis, vbds, vm) => mapValues(vdis, vdi => find(vbds, { VDI: vdi.id }))
   )
 
+  _getContainerBySr = createSelector(
+    () => this.props.hosts,
+    () => this.props.pool,
+    () => this.props.srs,
+    (hosts, pool, srs) =>
+      mapValues(srs, sr => {
+        const container = sr.$container
+        return container !== undefined && container === pool.id
```

Comment:
You don't need to check `container !== undefined` since `pool.id` is never undefined.